### ISSUE TITLE
Adobe Comment on 2.5.8 Target Size (Minimum)

### ISSUE THREAD

**Issue by awkawk** [w3c/wcag#1889]
Our main concern about this SC is that there are many government forms where an agency is required to replicate a paper form and the sizes are not sufficient. Perhaps if the essential exception added “or is legally required” then we can address that? 

It is also worth noting that common implementations of spin buttons where the up and down buttons combined are the same height as the control will not meet this SC.

**Comment by alastc** [comment]
Proposed response, as an alternative to PR #1954: 

<hr>

The group acknowledges that there are some circumstances where legally required forms of documents clash with accessibility requirements, and we try to minimise that overlap. However, in this case we do not think the text should be updated. It would be up to local legislators to make exceptions for this type of scenario.

**Comment by alastc** [comment]
Commenting on the spin button aspect, assuming it is an example like this [number input](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number).

![image](https://user-images.githubusercontent.com/216630/125972840-e20a5a03-c1e0-4ad6-8fa5-eb8cc277b617.png)

These seem to be user-agent based, so would fit under the exception being created for another issue.
